pulser-core/pulser/noise_model.py

```diff
@@ -76,6 +78,7 @@
     "p_false_pos",
     "p_false_neg",
     "amp_sigma",
+    "detuning_sigma",
```

Comment:
Since the detuning deviation is additive, does it actually make sense that it's probability like, ie in  [0,1]? Rather, it should just be a positive number, no?

Reply:
but why is `amp_sigma` probability like then? That is also just a positive number characterizing a distribution, not an actual probability. I put it there because `amps_sigma` was there.

Reply:
That being said, we do clip the sampled multiplicative factor to always be non-negative: https://github.com/pasqal-io/Pulser/blob/944b954ad6aee66d2385b1f0c8c0fbedfbd6343a/pulser-simulation/pulser_simulation/hamiltonian.py#L303